tests/test_listener.py

```diff
@@ -0,0 +1,91 @@
+"""This modules includes unit tests for the plugin."""
```

Comment:
If you decided to move these tests into a separate module - """... for the listener.""" ?